CONTRIBUTING.md

```diff
@@ -6,62 +6,106 @@ documentation, we greatly value feedback and contributions from our community.
 Please read through this document before submitting any issues or pull requests to ensure we have all the necessary
 information to effectively respond to your bug report or contribution.
 
+Table of contents:
+
+* [Reporting Bugs/Feature Requests](#reporting-bugsfeature-requests)
+* [Development](#development)
+    * [Finding contributions to work on](#finding-contributions-to-work-on)
+    * [Talk with us first](#talk-with-us-first)
+    * [Contributing via Pull Requests](#contributing-via-pull-requests)
+    * [Conventional Commits](#conventional-commits)
+* [Licensing](#licensing)
+
 ## Reporting Bugs/Feature Requests
```

Comment:
Somewhere in this file it might be useful to point users to the OJD spec repository if they have an OJD request. This repo is just for the Python implementation of the spec.

Reply:
I believe that's covered by the intro paragraph of the README. Do you think that some additional clarification should be added there?

Reply:
I could see someone working with the Python library, wishing it supported something, then raising an issue here that really would be a spec update. But maybe the README covers it. We can always add something later if people do raise issues in the wrong place.